src/Controller/Component/CsrfComponent.php

```diff
@@ -56,7 +56,7 @@ class CsrfComponent extends Component
         'cookieName' => 'csrfToken',
         'expiry' => 0,
         'secure' => false,
-        'httpOnly' => false,
+        'httpOnly' => true,
```

Comment:
The scenario is that the host page/application has a XSS issue, or a sketchy 3rd party script. In this case your CSRF cookie can be read and used to do CSRF attacks.